### PR TITLE
conditional styles, usr.SessionID + newsletter

### PR DESCRIPTION
**Pull request by anaisconce**

### CODE REVIEW

File: help/conditional-formatting.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+In this example, we have a list dog breeders who have raised champion thoroughbreds. Let's apply conditional formatting to the Breeder column based on the number of champion dogs. We would like to highlight in gold any breeders with more than 2 champions.
+
+![Writing conditional rules](../images/columns/first-conditional-rule.png)
+
+We would also like to highlight breeders with 1 or 2 champion dogs in blue, and 0 champion dogs in brown. Click `Add another rule` to add more conditional styles. 
+
+![Multiple conditional rules](../images/columns/multiple-conditional-rules.PNG)
+
+Note that Grist applies the rules in order. What would happen if we swapped the last two rules?
```
Comment (dsagal) [inline help/conditional-formatting.md:17]: I suggest a subheading here like "Order of rules", and a clarification after the first sentence:
```
Note that Grist applies all the rules in order. Styles applied by later rules will override those applied by earlier rules.

What would happen if we swapped the last two rules in the example above?
```

File: help/newsletters/2022-03.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+![Conditional Formatting](../images/newsletters/2022-03/conditional-formatting.gif)
+
+### Improved Column Type Guessing
+
+When you import or copy and paste data, Grist is now much better at parsing data and guessing column types. Phew! That means less data cleaning when importing new data! 🎉
+
+### New API Method for Add or Update
+
+We recently added a new method, PUT, to update an existing record, or, if it does not exist, add a new one. [Learn more.](https://support.getgrist.com/api/#tag/records)
```
Comment (dsagal) [inline help/newsletters/2022-03.md:110]: Can link more directly to `https://support.getgrist.com/api/#tag/records/paths/~1docs~1{docId}~1tables~1{tableId}~1records/put`. I would drop the `PUT` from the text though. It's not really the name of the method, it's only part of it. So just "We added a new method to update an existing record, etc." would be better.

File: help/newsletters/2022-03.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+Visit [our website](https://www.getgrist.com/sprouts-program/) to submit an inquiry! 
+
+## What's New
+
+### Conditional Formatting
+
+Conditional formatting is finally here! 🎊 In the creator panel in the column menu, under CELL STYLE you may now add conditional styles. Enter a formula and style the cell’s background and text color. You may add multiple conditional rules. [Learn more.](../conditional-formatting.md)
+
+![Conditional Formatting](../images/newsletters/2022-03/conditional-formatting.gif)
```
Comment (dsagal) [inline help/newsletters/2022-03.md:102]: Could you share this doc with me? I want to try and see if I can record this in a smaller space? I find that I need to scroll up and down just to see what's happening in the animation.

**Review by dsagal — COMMENTED** [review]
Thank you! For the newsletter, I'd like to suggest adding an extra section at the top for an above-the-fold summary of what's in this newsletter. Maybe a two-column thing:

## In this Newsletter:
Column 1
- Conditional formatting!
  [with a smallish cool screenshot]

Column 2:
- Announcing the Sprouts Program
  Let us help you build in Grist
- Custom Widgets Webinar
   Learn how to ...
- 3 new templates.

File: help/newsletters/2022-03.md (hunk 3 above)
Comment (anaisconce) [reply help/newsletters/2022-03.md:129]: Shared with you. It's the Champion Dogs doc that I butcher to test things and grab screenshots.

**Review by dsagal — APPROVED** [review]

**Pull request merged**